scripts/transformationsv2_config_translations.py

```diff
@@ -1,36 +1,71 @@
+import hashlib
+import json
+
+import numpy as np
+import pandas as pd
+
 from pathlib import Path
 
 import yaml
 
 from cognite.powerops.resync.config import ReSyncConfig, Transformation
 from cognite.powerops.resync.models._shared_v1_v2.cogshop_model import transformations_v2_transformer
 
-REPO_ROOT = Path(__file__).resolve().parent.parent
+
+def immutify_dictionary(d):
```

Comment:
type hints?

Reply:
This seems to also unpack `numpy` and `pandas` structures. Suggest returning a `MappingProxy`. See https://stackoverflow.com/questions/32720492/why-is-a-class-dict-a-mappingproxy

Reply:
Maybe rename to `to_sorted_hashable_dictionary`